Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/RootViewModel.swift

```diff
@@ -103,14 +153,14 @@ class RootViewModel: ObservableObject {
         }
     }
  
-    func handleResetButton() {
+    func reset() {
         Task {
             await UID2Manager.shared.resetIdentity()
             self.error = nil
         }
     }
     
-    func handleRefreshButton() {
+    func refresh() {
         Task {
             await UID2Manager.shared.refreshIdentity()
```

Comment:
Edge case, but what happens if you try to `refresh` while `self.error` is not null?